### DISCUSSION TITLE
Welcome to altair_ally Discussions!

### DISCUSSION THREAD

**Discussion by joelostblom** [joelostblom/altair_ally#1] [General]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

I am new to this discussion feature myself, but thought this would be a good opportunity to try it out.

**Comment by Jorsorokin** [comment]
Hi Joel, I stumbled upon this repo while also struggling to make a paired-down scattermatrix in altair. You've got some nice features in here already! Are you still actively working on this project? I'd love to contribute

**Reply by joelostblom** [reply]
Hello @Jorsorokin :wave: I am happy you found this repo and glad to hear you are finding it useful! I haven't been working on it in a while because I have been helping out with the main Altair library which is going through a larger upgrade right now. When that update is done, it might change how some things are done in `altair_ally`, so it might not be the best time for big changes until that happens. But I would love to hear your ideas and feedback in the meantime and if you have any specific contributions in mind maybe we can start planning for their implementation too.
